templates/zerver/app/index.html

```diff
@@ -41,6 +41,9 @@
     top: unset;
     bottom: 20px;
     }
+    body.color-scheme-automatic #app-loading{
+    color: hsl(240, 4%, 16%);
+    }
```

Comment:
How did you pick this color?  It doesn't seem to be used anywhere else in the app.  And why does this only set `color`, whereas the below sets `background-color` as well?

Reply:
Hi @timabbott, I tried different shades of black, and ended up with this shade. Should I change it?
This does not set the `background-color` because the background color for Automatic display settings is white by default, which is also what is desired.

Reply:
OK, but shouldn't we use the same color we use for black text in the app itself?

Reply:
Will change it ASAP. Thanks!

Reply:
The default color was HEX #333333. Have made the change, @timabbott. Kindly review.